packages/ui/src/pages/Home/LoaderWrapper.tsx

```diff
@@ -0,0 +1,25 @@
+import { Box, Grid } from '@mui/material'
+import { styled } from '@mui/material/styles'
+import React, { PropsWithChildren } from 'react'
+
+const LoaderWrapper = (props: PropsWithChildren) => {
```

Comment:
please add some comment in the PR description to what brought you to add this.

Reply:
khm code comment khm 😛